src/WebJobs.Script.WebHost/DependencyInjection/JobHostHostLifetime.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.DependencyInjection
+{
+    public class JobHostHostLifetime : IHostLifetime
```

Comment:
Is this for a JobHost or a ScriptHost? Also nothing in here seems to be specific to any JobHost/ScriptHost type? Where does this belong layering wise?